polyfills/Element/prototype/matches/config.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"browsers": {
 		"ie": "6 - *",
+		"ie_mob": "10 - *",
```

Comment:
Thanks, reading right now. How do you want me to approach this? Should I create another branch (not sure how I can revert this commit here) for the above? Or can you simply kill two birds in one shoot?